index.html

```diff
@@ -0,0 +1,200 @@
+      </ul>
+    </div>
+    <nav class="bg-white grid">
+      <div class="container f-row ai-center jc-between" id="navbar__container">
+        <div class="f-row jc-between" id="navbar__button-logo">
+        <div id="hamburger">
+          <i class="bi bi-list fs-40"></i>
+        </div>
+        <a href="#">
+          <img id="navbar__logo" src="./visuals/logo-jrwt.png" />
+        </a>
+      </div>
+          <ul class="f-col g-20 fs-20 fw-bold w-84 m-25" id="navbar__hamburguer__menulist">
+            <li class="btn-grey"><a class="w-100 ta-center" href="#">Home</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#dates">Dates</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#band">Band</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#">Music</a></li>
+            <li class="btn-orange"><a class="w-100 ta-center" href="#">Buy tickets</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="./about.html">About Us</a></li>
+          </ul>
+        <ul class="f-row g-70 jc-around ai-center fs-18 grey" id="navbar__menulist">
+          <li class="orange-hover"><a href="#">Home</a></li>
+          <li class="orange-hover"><a href="#dates">Dates</a></li>
+          <li class="orange-hover"><a href="#band">Band</a></li>
+          <li class="orange-hover"><a href="#">Music</a></li>
+          <li>
+            <button class="btn-orange fs-18 ff-1 ai-center">
+              <a href="#">Buy tickets</a>
+            </button>
+          </li>
+          <li class="orange-hover"><a href="./about.html">About Us</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="grid" id="front-page">
+      <div class="container f-col g-50" id="front-page__content">
+        <div class="f-col g-20">
+          <div class="f-col g-20 orange" id="front-page__title">
+            <h2 class="ff-3 fs-60">"The Unforgettable Odyssey"</h2>
+            <h1 class="ff-2 fs-100 fw-bold">JOSHUA</h1>
+            <h1 class="ff-2 fs-100 fw-bold w-50 ta-right">REDMAN</h1>
+            <h1 class="ff-2 fs-80 fw-lighter ta-right max-w-66" id="front-page__title__world-tour">
+              WORLD TOUR<br />2023
+            </h1>
+          </div>
+          <div class="bg-lighter box f-row border-white max-w-66" id="front-page__body-text">
+            <p class="fs-20">
+              Embark on a global musical journey with Joshua Redman and
+              experience the Legendary Saxophonist's Unforgettable Jazz
+              Odyssey across all Continents in 2023.
+            </p>
+          </div>
+        </div>
+        <div>
+          <h2 class="fs-40 grey">May 25, 2023 ~ Oct 31, 2023</h2>
+          <h3 class="fs-25 grey">
+            @ Lincoln Center for the Performing Arts, New York and much more
+          </h3>
+        </div>
+      </div>
+    </section>
+
+    <section class="grid bg-pattern-1 bg-dark white" id="dates">
```

Comment:
Great design @grauJavier 🤝

- [ ] As stated in the project requirements, kindly ensure your design follows the design guidelines as much as possible. kindly ensure your programs section contains five programs instead of four.
- Your design
![Your programs](https://github.com/grauJavier/capstone-project/assets/63917725/a6fca294-645b-4e8b-899f-0fc9add74f29)
- Given design
![Given program](https://github.com/grauJavier/capstone-project/assets/63917725/b55ad83a-9e42-4fb7-ac91-4c9d00f612ad)